examples/secondary-care/community/repeat-dispensing/nominated-pharmacy/clinical-practitioner/single-medication-request/1-Claim-Response-200_OK.xml

```diff
@@ -133,7 +130,8 @@
                       <seperatableInd value="true"/>
                       <pertinentDispensingEndorsement classCode="OBS" moodCode="EVN">
                         <code code="DE" codeSystem="2.16.840.1.113883.2.1.3.2.4.17.30"/>
-                        <value code="NDEC" codeSystem="2.16.840.1.113883.2.1.3.2.4.16.29"/>
+                        <text>£210.91,100ml,Specials Ltd,Lic12345678,BN12345678</text>
```

Comment:
Odd that the endorsements have changed when you've not touched the JSON request. Maybe this example was wrong before but it somehow wasn't causing test failures?